src/api/json/catalog.json

```diff
@@ -1617,11 +1617,12 @@
     },
     {
       "name": "ui5-manifest",
-      "description": "UI5/OPENUI5 descriptor file",
+      "description": "UI5 manifest file",
       "fileMatch": [
+        "manifest.json",
```

Comment:
There are already a few schemas with filematch "manifest.json". It's an overused file name. Please remove

Reply:
Hi @madskristensen , 

the filename manifest.json is used by a lot of UI5 applications to describe their bill of material (dependencies, datasources, routing information, ...). This file has been chosen at a point of time when the Web Manifest has been established and we are using namespaces to avoid conflicts in this file so that this can be technically mixed. 

An example how such a UI5 manifest file looks like can be seen here:
https://ui5.sap.com/#/topic/8f93bf2b2b13402e9f035128ce8b495f

How could we resolve this conflict? From our perspective, changing the file name is impossible as it has been manifested in the programming model since years. Is there an option to add additional match information? E.g. if the manifest.json is located aside of a Component.js or something similar? 

Thanks,
Peter

Reply:
If the file is in a folder with a name following a certain pattern, then you can do it.

Reply:
Thanks. Means if the file is always in `webapp/manifest.json` it could workout that way? 

Would it look like this?
```
      "fileMatch": [
        "webapp/manifest.json",
```

This could be used as a workaround for the time being. It would work at least for all our newer UI5 application projects following this project structure. But for projects having customized their project structure it will not work.

BTW: are there any considerations or options to identify the schema to be used via the file content? This would be also a nice alternative to overcome the clashes with the filename.

Reply:
Yes, that's exactly how it would work. Glad you found a way around this. `manifest.json` is by far the most problematic file name we've encountered